Realm/RLMSyncUser.h

```diff
@@ -214,29 +214,15 @@ NS_SWIFT_UNAVAILABLE("Use the full version of this API.");
  */
 - (void)revokePermission:(RLMSyncPermissionValue *)permission callback:(RLMPermissionStatusBlock)callback;
 
-// These permission APIs access Realms and Realm model objects representing
-// various permission states and actions, as well as standard Realm
-// affordances, to work with permissions. It is being deprecated in favor of
-// the `retrievePermissionsWithCallback:`, `applyPermission:callback:`, and
-// `revokePermission:callback:` APIs.
-#pragma mark - Realm Object-based Permissions API
-
 /**
  Returns an instance of the Management Realm owned by the user.
 
- This Realm can be used to control access permissions for Realms managed by the user.
- This includes granting other users access to Realms.
+ This Realm can be used to grant other users access to Realms managed by the current user.
+ 
+ @see `RLMSyncPermissionOffer`, `RLMSyncPermissionOfferResponse`
  */
 - (RLMRealm *)managementRealmWithError:(NSError **)error NS_REFINED_FOR_SWIFT;
```

Comment:
It's unfortunate that we weren't able to deprecate this in favor of the dynamic permission condition in time for 3.0. 🤷‍♂️